src/components/ChatRoomPage/Header.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useRecoilState } from 'recoil';
+import { currentUserState } from '../../state/atom';
+import userData from '../../db/userData.json';
+import styled from 'styled-components';
+import { RiArrowLeftSLine } from 'react-icons/ri';
+
+const Header = () => {
+  const [currentUser, setCurrentUser] = useRecoilState(currentUserState);
+
+  const toggleUser = () => {
+    if (currentUser.userId === 0) {
+      setCurrentUser(userData[1]); //나중에 수정 (현재 채팅 상대)
+    } else {
+      setCurrentUser(userData[0]);
+    }
+  };
+
+  return (
+    <HeaderBox>
+      <ExitBtn>
+        <RiArrowLeftSLine />
+      </ExitBtn>
+      {/* 채팅룸 나가기 버튼 */}
+      <UserInfo onClick={toggleUser}>
+        <UserImg
+          src={`${process.env.PUBLIC_URL}/Imgs/${currentUser.userImg}.jpg`}
+          alt={currentUser.userName}
+        />
+        <UserName>{currentUser.userName}</UserName>
+      </UserInfo>
+    </HeaderBox>
+  );
+};
+
+export default Header;
+
+const HeaderBox = styled.header`
+  display: flex;
+  align-items: center;
+
+  padding: 1rem;
+  padding-left: 0;
```

Comment:
```suggestion
  padding: 1rem 1rem 1rem 0;
```
요 부분은 이렇게 축약할 수 있을 것 같아요!